File: problems/sentimental/mario/more/mario.py
from cs50 import get_int

def main():
    # get variables for main
    height = ask()
    counter = 0
    row = 0

    # make hill
    while row < height:
        row += 1
        if counter != height:
            spaces = (height -1) - counter
            while spaces > 0:
                spaces -= 1
                print(" ", end="")

            hashes = 0
            while hashes <= counter:
                hashes += 1
                print("#", end="")

            print("  ", end="")

            hashes = 0
            while hashes <= counter:
                hashes += 1
                print("#", end="")

            print()
            counter += 1

def ask():
    while True:
        height = get_int("Height: ")
        if height >= 1 and height <= 8:
            break
    return height

if __name__ == "__main__":
    main()

File: problems/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

void hill(void);
int ask(void);

int main(void)
{
    // Calling Hill Function
    hill();
}



int ask(void)
{
    int height;

    do
    {
        height = get_int("Height: ");

    }
    while (height < 1 || height > 8);

    return(height);
}



void hill(void)
{
    int height = ask();
    if (height > 0 && height < 9)
    {
        int counter = 0;
        for (int row = 0; row < height; row++)
        {
            if (counter != height)
            {
                for (int spaces = (height -1) - counter; spaces > 0; spaces--)
                {
                    printf(" ");
                }
                for (int hashes = 0; hashes <= counter; hashes++)
                {
                    printf("#");
                }

                printf("  ");

                for (int hashes = 0; hashes <= counter; hashes++)
                {
                    printf("#");
                }

                printf("\n");
                counter++;
            }
        }
    }
}